recipes/newrelic/apm/php/debian.yml

```diff
@@ -143,59 +154,6 @@ install:
         - task: send_transaction
         - task: cleanup_temp_files
 
-    assert_pre_req:
-      cmds:
-        - |
-          if [ -z "$SUDO_USER" ]; then
-            echo -e "{{.RED}}This installation recipe for the New Relic PHP agent must be run under sudo.{{.NOCOLOR}}" >> /dev/stderr
-            exit 7
-          fi
-        - |
-          IS_SYSTEMCTL_INSTALLED=$(which systemctl | wc -l)
```

Comment:
Why was this check removed?  Both address specific cases where PHP installation can fail.  Without the check the user has no way of knowing why installation failed.